10up/publisher-media-kit
# Fix/62 Tab Title editable area fix

### Description of the Change

The tab title is now fixed to be editable from their own location as per the upstream tabs. To do so, the header of the tabs section (for example, "Our Rates") is moved on a separate line in the (backend area only) instead of displaying it inline with the tabs.

### Verification Process

- Go to the Media Kit page in the back end.
- Scroll to the "Our Rates" section.
- See the tab titles are editable from its own location like upstream.

### Checklist:






### Applicable Issues

Closes #62.

### Changelog Entry

> Fix - Tab Title is editable from below the tab
